Subject: [PATCH 1/1] Add -thor_kernel in version string
From: Volodya Ismailyan <nenie_iri@mail.ru>

Added -thor_kernel in the EXTRAVERSION string as the subject mandates it.

Signed-off-by: Volodya Ismailyan <nenie_iri@mail.ru>
---

--- Makefile	2023-10-10 22:04:19.952428876 +0400
+++ Makefile.orig	2023-10-11 12:09:12.592674650 +0400
@@ -2,7 +2,7 @@
 VERSION = 6
 PATCHLEVEL = 6
 SUBLEVEL = 0
-EXTRAVERSION = -rc4
+EXTRAVERSION = -thor_kernel
 NAME = Hurr durr I'ma ninja sloth
 
 # *DOCUMENTATION*
